recipe/meta.yaml

```
@@ -68,8 +68,8 @@ test:
   commands:
     - pip check
     - python -c "from polars import DataFrame"
-    - conda install -c conda-forge -y deltalake pyiceberg xlsx2csv
-    - pytest py-polars/tests -k "not ({{ tests_to_skip }})" {{ tests_modules_to_ignore }}
+    - conda install -c conda-forge -y deltalake pyiceberg xlsx2csv  # [py!=312]
```

```suggestion
    - conda install -c conda-forge -y deltalake pyiceberg xlsx2csv  # [py>=312]
```
I guess all python version >=3.12 can be skipped

---

fixed.